src/Utils/InitialConditions/init_2d_rising_bubble.jl

```diff
@@ -1,7 +1,7 @@
 """
-    init_dry_rising_bubble_2d(params, thermovar = :ρθ)
+    init_2d_rising_bubble(params, thermovar = :ρθ)
 
-    Dry rising bubble initial condition for 2D box benchmarking.
+    Rising bubble initial condition for 2D box benchmarking.
```

Comment:
looks dry to me... unless we want to add the moist option in the same file?

Reply:
Yes, we will have initial conditions that have different modes -> dry, etc.